src/Microsoft.HttpRepl.IntegrationTests/Commands/SetSwaggerCommandTests.cs

```diff
@@ -0,0 +1,269 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+using System.Linq;
+using System.Net.Http;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.HttpRepl.Commands;
+using Microsoft.HttpRepl.IntegrationTests.Mocks;
+using Microsoft.HttpRepl.Resources;
+using Microsoft.Repl.Parsing;
+using Xunit;
+
+namespace Microsoft.HttpRepl.IntegrationTests.Commands
+{
+    public class SetSwaggerCommandTests : CommandHelper<SetSwaggerCommand>
+    {
+        public SetSwaggerCommandTests()
+          : base(new SetSwaggerCommand())
+        {
+        }
+
+        [Fact]
+        public void GetHelpDetails_WithInvalidParseResultSection_ReturnsNull()
+        {
+            string result = GetHelpDetails(parseResultSections: "section1");
```

Comment:
I'm really disliking this CommandHelper style, as it completely breaks AAA (Arrange/Act/Assert).  From reading these tests, I don't know what has been arranged (1) and I don't know what action is being taken (2).  In order to understand this test, I have to jump to another body of code, which takes me out of the context of what's being tested.

For point 1:
- In a positive test (i.e. happy path), I'm willing to assume a sane test default, and that the details don't matter.
- In a negative test (e.g. given an invalid input, assert error handling), I'd really like to see in the test body what the special-case arrangement is for this scenario.  It needs to be very clear what makes the input invalid.

Point 2 is actually the more important one.  Even if I assume the arrangement is correct in all cases (positive/negative/edge), I'm not actually sure what's being tested here without jumping through sources.  I'm more willing to accept helpers for arranging, but I prefer that the Act part of a test be clear and explicit - what is being called, and with what paramemters, because it helps me assess the asserts better.

To make this more readable, I'd like to see something like this:

        [Fact]
        public void GetHelpDetails_WithFirstParseResultSectionNotEqualToName_ReturnsNull()
        {
            ArrangeGetHelpDetailsInputs(commandText: "section1", out ShellState s, out HttpState h, out ICoreParseResult p);
            var subject = new SetSwaggerCommand(); // clearly define the subject being tested

            string result = subject.GetHelpDetails(s, h, p); // clearly define the action being tested

            Assert.Null(result);
        }

Reply:
As discussed offline, will send out another pr for this change.